docs/getting-started.adoc

```diff
@@ -0,0 +1,196 @@
+= Getting Started
+
+This document walks you through the development and deployment of a Node.js application to an OpenShift cluster using `odo`. It guides you on how to push new changes to your application and on other useful features of `odo`.
+
+== Developing and deploying an application to an OpenShift cluster
+
+. link:/docs/installation.md[Install the `odo` binary].
+. Log into an OpenShift cluster:
+
+* If you lack a local development cluster, we recommend using
+https://docs.openshift.org/latest/minishift/getting-started/installing.html[Minishift]
+to deploy a development OpenShift cluster.
++
+.. Run Minishift:
++
+[source,bash]
+----
+$ minishift start
+----
+.. Log into the OpenShift cluster:
++
+[source,bash]
+----
+$ odo login -u developer -p developer
+----
++
+NOTE: In order to make full use of `odo` functionality, it is recommended to enable the OpenShift https://docs.openshift.com/container-platform/3.11/architecture/service_catalog/index.html[service catalog]. Use `minishift` 1.30 or latest to enable this: `sh   $ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--enable=*,service-catalog,automation-service-broker,template-service-broker"`
+
+* If you do not use Minishift, `odo` automatically works with any OpenShift cluster you are currently logged into. To log into the OpenShift cluster run:
++
+[source,bash]
+----
+$ odo login -u developer -p developer
+----
+
+. An application is an umbrella that comprises of all the components (microservices) you build. Create an application:
++
+[source,bash]
+----
+$ odo app create nodeapp
```

Comment:
as per the new workflow, we don't create the app.
cc @kadel @anmolbabu @cdrage

Reply:
@surajnarwade can you please let me know what is the new workflow so that I can work it? Is it a minor change or a complete rehaul of the workflow? Or do we want to create separate issue for it and get this merged now?

Reply:
Yes @Preeticp there is no app creation now and a bunch of other commands are also deprecated
https://docs.google.com/document/d/1RKva9lvIR6dLmDm2tlqmp8V4nFjlrXdulOQ4FSmLkr8/edit?ts=5c715ffd#heading=h.eqonbgz6nece is the workflow doc

Reply:
All the commands that are striked out are expected to be deprecated

Reply:
@anmolbabu I have been working on the doc set only since Monday (1st April), and am not familiar enough with the commands to understand it after reading the doc. Based on what you and suraj are saying, do I need to delete only this command or others too in the PR. Also I need to get this and the other docs in by the coming Monday so very short on time.
Request you to please review the PR and point out specific instructions which need to be deleted based on the current situation and not the expected situation. Thanks in advance.

Reply:
@surajnarwade ^^